integration/tests/cook/test_master_slave.py

```diff
@@ -24,6 +25,10 @@ def setUp(self):
         self.slave_url = type(self).slave_url
         self.logger = logging.getLogger(__name__)
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
     def test_get_queue(self):
         job_uuid, resp = util.submit_job(self.master_url, constraints=[["HOSTNAME",
```

Comment:
Is it possible to use a different constraint that causes jobs to not be scheduled and avoid the issue of launching hosts that never get used?

Reply:
I changed this test to build up a queue